packages/gatsby/src/recipes/README.md

```diff
@@ -0,0 +1,137 @@
+# Gatsby Recipes
+
+Gatsby Recipes is framework for automating common Gatsby tasks. Recipes are MDX
+files which when run by our interpretor, perform common actions like installing
+NPM packages, installing plugins, creating pages, etc.
+
+It's designed to be extensible so new capabilities can be added which allow
+Recipes to automate more things.
+
+We chose MDX to allow for a literate programming style of writing recipes which
+enables us to port our dozens of recipes from
+https://www.gatsbyjs.org/docs/recipes/ as well as in the future, entire
+tutorials.
+
+[Read more about Recipes on the RFC](https://github.com/gatsbyjs/gatsby/pull/22610)
+
+There's an umbrella issue for testing / using Recipes during its incubation stage.
+
+Follow the issue for updates!
+
+https://github.com/gatsbyjs/gatsby/issues/22991
+
+## How to write a recipe
+
+Let's write our first Recipe to setup Emotion on a Gatsby site.
+
+```mdx
+# Setup Gatsby with Emotion
+
+[Emotion](https://emotion.sh/) is a powerful CSS-in-JS library that supports both inline CSS styles and styled components. You can use each styling feature individually or together in the same file.
+
+---
+
+Install necessary NPM packages
+
+<NPMPackage name="gatsby-plugin-emotion" />
+<NPMPackage name="@emotion/core" />
+<NPMPackage name="@emotion/styled" />
+
+---
+
+Install the Emotion plugin in gatsby-config.js
+
+<GatsbyPlugin name="gatsby-plugin-emotion" />
+
+---
+
+Sweet, now it's ready to go.
+
+Let's also write out an example page you can use to play
+with Emotion.
+
+<File
+  path="src/pages/emotion-example.js"
+  content="https://gist.githubusercontent.com/KyleAMathews/323bacd551df46e8e7b6146cbf827d0b/raw/5c60f168f30c505cff1ff2433e69dabe27ae9738/sample-emotion.js"
+/>
+
+---
+
+Read more about Emotion on the official Emotion docs site:
+
+https://emotion.sh/docs/introduction
+```
+
+Let's try running out this Recipe!
+
+First create a new Hello World Gatsby site:
+
+`gatsby new try-emotion https://github.com/gatsbyjs/gatsby-starter-hello-world`
+
+Meanwhile also upgrade the global gatsby-cli package to use the one with new recipes command.
```

Comment:
Can we put this first? Separate out the instructions that allow for recipes from what we're adding to the example.